module/VuFind/src/VuFind/RecordTab/Factory.php

```diff
@@ -262,6 +262,13 @@ public static function getReviews(ServiceManager $sm)
     public static function getUserComments(ServiceManager $sm)
     {
         $capabilities = $sm->getServiceLocator()->get('VuFind\AccountCapabilities');
-        return new UserComments('enabled' === $capabilities->getCommentSetting());
+        $config = $sm->getServiceLocator()->get('VuFind\Config')->get('config');
+        $useRecaptcha = isset($config->Captcha) && isset($config->Captcha->forms)
+            && (trim($config->Captcha->forms) === '*'
+            || strpos($config->Captcha->forms, 'userComments'));
```

Comment:
These three lines are a copy of logic from `/Controller/Plugin/Recaptcha->validate()`. I couldn't find a way to use that code here in the Factory though.